src/layouts/base/index.tsx

```diff
@@ -19,32 +21,37 @@ export type BaseLayoutProps = PropsWithChildren<{
 export function BaseLayout({ children, title }: BaseLayoutProps) {
   useHtmlDarkMode();
   const theme = useTheme();
-  const { t } = useTranslation();
+  const { t, language } = useTranslation();
 
   const siteTitle = usePageTitle(title);
 
+  // used for calendards and date pickers, transform to locale supported by react-aria
+  const i18nLocale = language === Language.English ? 'en-GB' : 'nb-NO';
+
   return (
-    <div className={style.wrapper}>
-      <Head>
-        <title>{siteTitle}</title>
-        <meta
-          name="description"
-          content={t(CommonText.Layout.meta.defaultDescription)}
-        />
-        <link rel="icon" href="/assets/colors/icons/favicon.svg" />
-        <meta
-          name="theme-color"
-          content={theme.static.background.background_accent_0.background}
-        />
-      </Head>
-
-      <OpenGraphBase title={siteTitle} />
-
-      <PageHeader />
-
-      <main className={style.main}>{children}</main>
-
-      <Footer />
-    </div>
+    <I18nProvider locale={i18nLocale}>
```

Comment:
Can't this provider be colocated with the other context providers? I guess it's because of `useTranslation` and how it works?

Reply:
It cannot because it needs to access the outer scope of the contexts yes. So in that case we'd need to introduce another layer of components, which would be for that sole purpose, and then I think this solution is as good as any